shared/search/container.js

```diff
@@ -36,6 +36,22 @@ const mapStateToProps = (state: TypedState) => {
   }
 }
 
+const mobileOnClickUserInGroup = (dispatch, user) => {
```

Comment:
more complicated so pulled it up. If its a normal profile do the old stuff. if its an external one push a nonProfile